erpnext/accounts/report/accounts_receivable/accounts_receivable.py

```diff
@@ -378,6 +378,14 @@ def prepare_conditions(self, party_type):
 				conditions.append("""party in (select parent
 					from `tabSales Team` where sales_person=%s and parenttype = 'Customer')""")
 				values.append(self.filters.get("sales_person"))
+
+		if party_type_field=="supplier":
+   		    if self.filters.get("supplier_type"):
+		        suptype = self.filters.get("supplier_type")
+    		        conditions.append("""party in (select name from tabSupplier
+		                where supplier_type=%s)""")
```

Comment:
`supplier_type` field also renamed to `supplier_group`

Reply:
thats ok i have applied this requests and going to apply the pull request again